src/bindings/jna/libelektra/src/main/java/org/libelektra/exception/KeySetReleasedException.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Indicates that an already released {@link KeySet} has been accessed
  */
-public class KeySetReleasedException extends RuntimeException
+public class KeySetReleasedException extends IllegalStateException
```

Comment:
Do we really need our own exception for this? Why would an API user want to catch this?

Reply:
I think you are right, we could do without KeySetReleasedException and KeyReleasedException, throwind an IllegalStateException instead.